docs/en/resources/tools/bigquery/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Bigquery"
+type: docs
+weight: 1
+description: > 
+  Bigquery Tools.
```

Comment:
nit: slightly longer descriptions would be nice. Maybe:

"A list of tools that work with BigQuery source's".